src/components/GamePlay.js

```diff
@@ -26,7 +27,7 @@ const sanitizeInputs = (input) => {
     return input;
   }
   const trimmedInput = input.trim();
-  const onlyStrings = trimmedInput.replace(/[^a-zA-Z ]/g, '');
+  const onlyStrings = trimmedInput.replace(/[^a-zA-Z ]/g, '').toLowerCase();
```

Comment:
Be sure to make these changes on the correct branch. Adding a string method to this func should only be done when working on this feature.

Reply:
Forgot to add to the prior commit for this component and wanted to make the change while I was already working on this component. Will be sure to switch branches next time.